docs/releasenotes/1.3_migration_guide.md

```diff
@@ -0,0 +1,93 @@
+
+In FireFly v1.2.0 to create one of the affected resources and publish it to other parties, a `POST` call would be made to its respective API route and the broadcast would happen immediately. To achieve the same behaviour in FireFly v1.3.0, there are 2 options for all impacted resources, either providing a query parameter at creation to signal immediate publish, or a subsequent API call to publish the resources.
+
+> **NOTE**: All of the impacted resources can be deleted locally using `DELETE` APIs, however, once the resources has been published, it is no longer possible to delete.
+
+### Contract interfaces
+
+Previously, to create a contract interface a `POST` call would be made to `/contracts/interfaces` and the interface would be broadcasted to all other namepsaces. In FireFly v1.3.0, this same call can be made with the `publish=true` query parameter, or a subsequent API call can be made on an unpublished interface on `POST /contracts/interfaces/{name}/{version}/publish` specifying the name and version of the interface.
+
+[For an exact view of the changes to contract interfaces, see PR #1279.](https://github.com/hyperledger/firefly/pull/1279)
+
+### Contract APIs
+
+Previously, to create a contract API a `POST` call would be made to `/apis` and the API would be broadcasted to all other namepsaces. In FireFly v1.3.0, this same call can be made with the `publish=true` query parameter, or a subsequent API call can be made on an unpublished API on `/apis/{apiName}/publish` specifying the name of the API.
+
+[For an exact view of the changes to contract APIs, see PR #1322.](https://github.com/hyperledger/firefly/pull/1322)
+
+### Token pools
+
+Previously, to create a token pool a `POST` call would be made to `/tokens/pools` and the token pool would be broadcasted to all other namepsaces. In FireFly v1.3.0, this same call can be made with the `publish=true` query parameter, or a subsequent API call can be made on an unpublished token pool on `/tokens/pools/{nameOrId}/publish` specifying the name or ID of the token pool.
+
+[For an exact view of the changes to token pools, see PR #1261.](https://github.com/hyperledger/firefly/pull/1261)
+
+## Event stream considerations
+
+### Single event stream per namespace
+
+In this release, the model for event streams in a multi-party network has fundamentally changed. Previously, there was a single event stream for all members in the network, in FireFly v1.3.0 there is now a single event stream per namespace in the network. The migration to individual event streams promotes high-availability capability but is not itself a breaking change, however the ID format for event streams has changed.
+
+Event streams now follow the format `<plugin_topic_name>/<namespace_name>`. For example, an event stream for the default namespace with a plugin topic of 0 would now be: `0/default`.
+
+[For an exact view of the changes, see PR #1388.](https://github.com/hyperledger/firefly/pull/1388)
```

Comment:
Two other notes here that should be added:
1. The migration from FireFly 1.2.x -> 1.3 will result in a rebuilding of the existing event streams. This means that connectors will replay past events to FireFly, but FireFly will automatically de-duplicate them by design so this is a safe operation.
2. These changes should result in no impact to end users of FireFly. They are merely noted here in the release notes because they are a significant architectural change to the relationship between namespaces, plugins, and connectors.

Reply:
Restructured with changes in https://github.com/hyperledger/firefly/pull/1461/commits/c0b0901e27687a47201733d0a052a59db63fa5de